src/events/dns.js

```diff
@@ -1,12 +1,9 @@
-const uuid4 = require('uuid4');
 const dns = require('dns');
 const shimmer = require('shimmer');
 const utils = require('../utils.js');
 const tracer = require('../tracer.js');
-const serverlessEvent = require('../proto/event_pb.js');
 const eventInterface = require('../event.js');
-const errorCode = require('../proto/error_code_pb.js');
-const { isBlacklistURL } = require('.././helpers/events');
+const { isBlacklistURL, initializeEvent } = require('.././helpers/events');
```

Comment:
initializeEvent should be under `eventInterface`

Reply:
moved.